.eslintrc.json

```diff
@@ -24,7 +34,97 @@
     {
       "files": ["*.ts", "*.tsx"],
       "extends": ["plugin:@nx/typescript"],
-      "rules": {}
+      "rules": {
+        "testing-library/prefer-screen-queries": 0,
+        "comma-dangle": [
+          "error",
+          {
+            "arrays": "always-multiline",
+            "exports": "only-multiline",
+            "functions": "never",
+            "imports": "only-multiline",
+            "objects": "only-multiline"
+          }
+        ],
+        "eol-last": "error",
+        "import/order": [
+          "warn",
+          {
+            "groups": [
+              "builtin",
+              "external",
+              "index",
+              "sibling",
+              "parent",
+              "internal"
+            ]
+          }
+        ],
+        "jsx-quotes": ["warn", "prefer-double"],
+        "max-len": [
+          "warn",
+          {
+            "code": 120,
+            "ignorePattern": "d=\"([\\s\\S]*?)\""
+          }
+        ],
+        "no-console": "warn",
+        "no-debugger": "warn",
+        "no-duplicate-imports": "warn",
+        "no-unused-vars": "off",
+        "object-curly-spacing": ["warn", "always"],
+        "quotes": ["warn", "double"],
+        "react/jsx-curly-spacing": [
+          "error",
+          {
+            "allowMultiline": true,
+            "children": {
+              "when": "always"
+            },
+            "spacing": {
+              "objectLiterals": "always"
+            },
+            "when": "always"
+          }
+        ],
+        "react/jsx-filename-extension": [
+          "error",
+          {
+            "extensions": [".js", ".jsx", ".ts", ".tsx"]
+          }
+        ],
+        "react/jsx-sort-props": [
+          2,
+          {
+            "callbacksLast": true,
+            "shorthandFirst": false,
+            "shorthandLast": true,
+            "ignoreCase": true,
+            "noSortAlphabetically": false
+          }
+        ],
+        "react/prop-types": "warn",
+        "semi": "warn",
+        "sort-imports": [
+          "warn",
+          {
+            "ignoreCase": false,
+            "ignoreDeclarationSort": true,
+            "ignoreMemberSort": false
+          }
+        ],
```

Comment:
I tested out another import sorter: `“simple-import-sort”` as noted in https://medium.com/aeturnuminc/eslint-configuration-for-react-formatters-and-rules-8ea0a29dd97

I preferred the layout and spacing of the `“simple-import-sort”`, I will see if the same options can be applied for our current `"sort-imports"` otherwise might recommend moving to the referenced sorter.